Mathlib/Topology/Category/TopCat/Limits/Products.lean

```diff
@@ -224,7 +224,7 @@ theorem range_prod_map {W X Y Z : TopCat.{u}} (f : W ⟶ Y) (g : X ⟶ Z) :
   · rintro ⟨y, rfl⟩
     simp_rw [Set.mem_inter_iff, Set.mem_preimage, Set.mem_range]
     -- sizable changes in this proof after #13170
```

Comment:
This is the same ref that's mentioned in the other rw -> erw notes; can (and should) we try to go back to the older proof?

Reply:
We cannot, unfortunately: `simp` can't use `comp_apply` here. (To be very specific, the `CoeSort` instance is not reducibly defeq between `TopCat` and `ConcreteCategory`, and so they don't end up in the same discrimination tree key.)